ilastik/shell/gui/ilastikShell.py

```diff
@@ -626,6 +631,18 @@ def openFileAndCloseStartscreen(self, path):
         #del self.startscreen
         self.openProjectFile(path)
 
+    def _createDocButton(self):
+        pushButton = QMenu("&Check the docs!", self)
+        pushButton.setObjectName("docs_menu")
+        aboutIlastikPushAction = pushButton.addAction("&ilastik 1.0.0 developer documentation")
+        aboutIlastikPushAction.triggered.connect(self.on_pushButton_clicked)
+        return pushButton
+
+    def on_pushButton_clicked(self):
+        webbrowser.open('http://ilastik.org/ilastik/', new=2)
```

Comment:
I would suggest to link to the user documentation instead of the developer one (http://ilastik.org/documentation/)

Reply:
thanks @FynnBe for review and suggestion, today i will link to the specific user documentation, as in issue there is no specific link given so i thought to link the developer one, i will correct and give the pr request soon